### PR TITLE
Allow passing domains with zoneFile

### PR DESCRIPTION
**Pull request by qlonik**
This PR addresses #155 by allowing for optional `domains` field in `zoneFile` object. It also allows passing more than one zoneFile using `zoneFiles` array in a way similar to coredns: https://github.com/coredns/helm/blob/4a9f0dd13f08ff41ce768b11a830e48e54ef8b1d/charts/coredns/templates/configmap.yaml#L40-L42

### CODE REVIEW

File: charts/k8s-gateway/templates/configmap.yaml [hunk 1]
```diff
@@ -36,9 +36,15 @@ data:
         }{{ end }}
         {{- end }}
         {{- if .Values.zoneFile }}
```
Comment (networkop) [inline charts/k8s-gateway/templates/configmap.yaml:38]: I believe this needs to go

File: charts/k8s-gateway/templates/configmap.yaml [hunk 2]
```diff
@@ -36,9 +36,15 @@ data:
...
-        file /etc/coredns/{{ .Values.zoneFile.filename }}
+        file /etc/coredns/{{ .Values.zoneFile.filename }}{{ if .Values.zoneFile.domains }} {{ .Values.zoneFile.domains }}{{ end }}
+        {{- end }}
```
Comment (networkop) [inline charts/k8s-gateway/templates/configmap.yaml:40]: I believe all this needs to go?

**Comment by qlonik** [comment]
updated it

**Comment by networkop** [comment]
lgtm. thanks!

**Pull request merged**